grpc-api/server/docker/build.gradle

```diff
@@ -113,4 +115,29 @@ dockerCreateDockerfile.dependsOn grpcApiBase
 assemble.dependsOn dockerBuildImage
 dockerBuildImage.outputs.upToDateWhen { t -> return Docker.isImageUpToDate(t) }
 
+// Create a simple file for docker/build-push-action to reference, tag, and publish
+tasks.register('publishDockerfile', Dockerfile) {
+  def input = dockerBuildImage.imageIdFile
+  def output = layout.buildDirectory.file('publish/Dockerfile')
+
+  it.inputs.file input
+  it.outputs.file output
+
+  it.destFile.set output
```

Comment:
i think the outputs.file is not needed above when you have this line here, as destFile is marked as an output already